docs/user/field-selectors.md

```diff
@@ -0,0 +1,149 @@
+# Field Selectors
+
+Field selectors filter a list of Flight Control objects based on specific resource field values.<br>
+They follow the same syntax, principles, and support the same operators as Kubernetes Field and Label selectors.
```

Comment:
I'd add something along of the lines of the suggestion, given we next describe more operators than only those supported by Kubernetes.

```suggestion
They follow the same syntax, principles, and support the same operators as Kubernetes Field and Label selectors. Some additional operators are also supported to allow for more advanced search use cases.
```

Reply:
Change to: 
They follow the same syntax, principles, and support the same operators as Kubernetes Field and Label selectors, with additional operators available for more advanced search use cases.